lib/ansible/plugins/lookup/sequence.py

```diff
@@ -227,7 +227,7 @@ def run(self, terms, variables, **kwargs):
                 self.set_options(direct=kwargs)
                 try:
                     if not self.parse_simple_args(term):
-                        self.parse_kv_args(parse_kv(term))
+                        self.parse_kv_args(parse_kv(term, populate_raw_params=False))
```

Comment:
Can you add an integration test to verify the error handling for this?

Reply:
was already on it!

Reply:
it seems we would error out later on either way, this just fails faster and slightly changes the message.